SpringBootCrudApi/src/main/java/com/cengenes/configuration/api/controller/CustomerController.java

```diff
@@ -0,0 +1,107 @@
+package com.cengenes.configuration.api.controller;
+
+import com.cengenes.configuration.api.entity.customer.Customer;
+import com.cengenes.configuration.api.entity.customer.CustomerResponse;
+import com.cengenes.configuration.api.entity.response.OperationResponse;
+import com.cengenes.configuration.api.repository.CustomerRepository;
+import com.cengenes.configuration.api.service.AuthService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.domain.Page;
+import org.springframework.data.domain.Pageable;
+import org.springframework.http.MediaType;
+import org.springframework.transaction.annotation.Transactional;
+import org.springframework.web.bind.annotation.*;
+import javax.servlet.http.HttpServletResponse;
+
+@RestController
+@Transactional

```

Comment:
@Transactional annotation must be inside service layer. Your business can decide transaction finished or not.